pimd/pim_nb_config.c

```diff
@@ -235,12 +235,6 @@ static int pim_rp_cmd_worker(struct pim_instance *pim, pim_addr rp_addr,
 
 	result = pim_rp_new(pim, rp_addr, group, plist, RP_SRC_STATIC);
 
-	if (result == PIM_RP_NO_PATH) {
```

Comment:
I disagree.  It's an operational warning that what is configured will not work at all.  No different than all the other places we warn the operator that something is not quite right yet.

Reply:
>  It's an operational warning that what is configured will not work at all.

It will work after your unicast routing finishes booting up. We accept "not yet valid" config _everywhere_ else without saying anything about it. Or can you point me at some other place where we do this?

Reply:
It's especially bad because you will get this warning during initial configuration load at bootup, it's really misleading if you're trying to hunt down an actual issue…

Reply:
dropped commit to keep the remainder moving